catalog/core/catalog-core-impl/pubsub/src/main/java/ddf/catalog/pubsub/criteria/contextual/CaseSensitiveContextualAnalyzer.java

```diff
@@ -68,9 +68,7 @@
      * Specifies whether deprecated acronyms should be replaced with HOST type. See
      * {@linkplain https://issues.apache.org/jira/browse/LUCENE-1068}
      */
-    private final boolean replaceInvalidAcronym, enableStopPositionIncrements;
-
-    private final Version matchVersion;
```

Comment:
@vinamartin that is a valid concern. I think the comment on line 69 is interesting. For `matchVersion` it is getting passed around in parameters where it is needed. The `replaceInvalidAcronym` variable seems to just be set there and not used again. Could be carryover from previous refactoring. @pklinef might recall something else on it.